### PR TITLE
Feature/views

### PR DESCRIPTION
**Pull request by mistermaxis**
### Tasks Completed:
- Create users view
- Create singular user view
- Create posts view
- Create singular post view

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @posts = Post.order('id Asc')
+    @post = Post.first
+  end
```
Comment (Gambit142) [inline app/controllers/posts_controller.rb:5]: - [ ] (Required) The index action for the post controller should show all posts for a single user and not all the posts in the database. This query gets all the posts for all users. 

_**Hint:**_ You should get the user and display all posts for that user. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @mistermaxis 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1 +1,11 @@
...
+<% @users.each do |user| %>
+  <div class="flex grow gap-x-3 items-center">
+    <div class="h-20 w-20 bg-black border-black border-2"></div>
+    <div class="flex grow justify-between border-black border-2">
+      <div class="text-4xl p-5"><%= link_to user.name, user_path(user.id) %></div>
+      <span class="self-end p-5">Number of Posts: <%= user.posts_counter || 0 %><span>
+    </div>
+  </div>
+  <% end %>
+</div>
```
[the remark this answers is not in the document]
Comment (mistermaxis) [reply app/views/users/index.html.erb:11]: Thank you for the suggestion. That is true, I still haven't looked into that yet, but I will.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Max, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request closed without merging**